esrally/mechanic/telemetry.py

```diff
@@ -795,35 +799,53 @@ def attach_to_node(self, node):
     def on_benchmark_start(self):
         if self.process is not None:
             self.process_start = sysstats.process_io_counters(self.process)
+            read_bytes = 0
+            write_bytes = 0
+            io.ensure_dir(self.log_root)
+            log_file = "%s/%s-%s.io" % (self.log_root, self.car.safe_name, self.node.node_name)
```

Comment:
This for example produces something like:

```
.rally/benchmarks/races/2019-07-04-15-05-34/rally-node-0/telemetry/defaults-rally-node-0.io
```

I don't think `self.car.safe_name` (in this case `defaults`) provides value in the filename, in which case you don't even need to pass car in the constructor. WDYT?

Reply:
I agree, but I was trying to be consistent with what we do fo gcstats and jfr and other telemetry files. So not sure whether to go for simplicity or consistency. What's your preference?

Reply:
Contrary to other telemetry devices this one is not meant to be inspected by the user. Instead this is a temporary state file that should also be removed when the benchmark ends (i.e. in `on_benchmark_stop`). Hence, I'd go for simplicity here.